### DISCUSSION TITLE
Total ascent data from Treadmill

### DISCUSSION THREAD

**Discussion by dylanebner** [philosowaffle/peloton-to-garmin#475] [General]
Is it possible to get the total ascent field in Garmin populated with the Elevation data from Peloton?

**Comment by philosowaffle** [comment]
Possibly. I currently only have one sample set of data from a Tread workout. It does have an [elevation metric](https://github.com/philosowaffle/peloton-to-garmin/blob/master/src/UnitTests/Data/p2g_workouts/tread_run_workout.json#L12548) in it.

**Reply by dylanebner** [reply]
do you need more samples? I could send you a few of mine.
